ci/prepare-unity.sh

```diff
@@ -0,0 +1,64 @@
+#!/usr/bin/env bash
+
+### This script is used to setup Improbable's internal build machines.
+### If you don't work at Improbable, this may be interesting as a guide to what software versions we use for our
+### automation, but not much more than that.
+
+set -e -u -x -o pipefail
+
+cd "$(dirname "$0")/../"
+
+source ci/includes/pinned-tools.sh
+source ci/includes/profiling.sh
+
+# Only run inside TeamCity
+if [ -z "${TEAMCITY_CAPTURE_ENV+x}" ]; then
+  exit 0
```

Comment:
needs an error message

Reply:
How is: "This script is only to be run on Improbable CI" ?